add_up.rb

```diff
@@ -0,0 +1,14 @@
+3.times do 
+    print "Please enter a positive, whole number. "
+  answer = gets.chomp.to_i
+    def add_up(i)
```

Comment:
You probably want to move this above the `3.times` loop because there's no point in redefining this at every iteration. It doesn't cost much, but still.